ui/ui-components/pages/model/[modelId]/overview.tsx

```diff
@@ -0,0 +1,144 @@
+import { GetServerSideProps, NextPage } from "next";
+import Head from "next/head";
+import { useMemo } from "react";
+import { Col, Container, ListGroup, ListGroupItem, Row } from "react-bootstrap";
+import ModelOverviewCard from "../../../components/model/overview/ModelOverviewCard";
+import ModelOverviewDestinations from "../../../components/model/overview/ModelOverviewDestinations";
+import ModelOverviewGroups from "../../../components/model/overview/ModelOverviewGroups";
+import ModelOverviewPrimarySource from "../../../components/model/overview/ModelOverviewPrimarySource";
+import ModelOverviewSchedules from "../../../components/model/overview/ModelOverviewSchedules";
+import ModelOverviewSecondarySources from "../../../components/model/overview/ModelOverviewSecondarySources";
+import PageHeader from "../../../components/PageHeader";
+import { GrouparooModelContextProvider } from "../../../contexts/grouparooModel";
+import { UseApi } from "../../../hooks/useApi";
+import { Actions, Models } from "../../../utils/apiData";
+
+interface Props {
+  model: Models.GrouparooModelType;
+  primarySource?: Models.SourceType;
+  secondarySources: Models.SourceType[];
+  groups: Models.GroupType[];
+  schedules: Models.ScheduleType[];
+  destinations?: Models.DestinationType[];
+}
+
+const Page: NextPage<Props & { ctx: any; errorHandler: any }> = ({
```

Comment:
This could be more precisely named, maybe `ModelPage`?

Reply:
This is the general naming convention for all pages in the project

Reply:
Ah, okay.